src/tests/ftest/launch.py

```diff
@@ -751,11 +751,6 @@ def get_test_files(test_list, args, yaml_dir, vmd_flag=False):
         yaml_file = replace_yaml_file("{}.yaml".format(base), args, yaml_dir, vmd_flag)
         test_file["yaml"] = yaml_file
 
-        # Set enable_vmd: true in the daos_server yaml if there are VMD devices on the host
-        # regardless of whether they are being specified in the server config file to avoid
-        # failures in the server start-up NVMe scan.
-        test_file["env"] = {"DAOS_ENABLE_VMD": "True" if vmd_flag else "False"}
-
```

Comment:
This code is required for CI.  When we run on non-VMD enabled clusters we need launch.py to correctly set the enable_vmd/disable_vmd variable.

Reply:
can you explain why that is still needed, with this change daos_server shouldn't need-to-know whether the bdev_list PCI addresses are VMD endpoints or not, the only time we should need to set disable_vmd is in the case where VMD is enabled on the system but we want to use non-VMD enabled SSDs. as far as I know that shouldn't happen in CI.

Reply:
Ok, then I recant my objection.  Thanks for the clarification.